stroom-core-client/src/main/java/stroom/document/client/DocumentPluginEventManager.java

```diff
@@ -759,8 +760,13 @@ private void addModifyMenuItems(final List<Item> menuItems,
                     DocumentPermissionNames.OWNER,
                     true);
             if (ownedItems.size() == 1) {
-                menuItems.add(new Separator(10));
-                menuItems.add(createPermissionsMenuItem(ownedItems.get(0), 11, true));
+                if (securityContext.hasAppPermission(PermissionNames.VIEW_SYSTEM_INFO_PERMISSION)) {
```

Comment:
system info permission is the permission to use the system info API.  There is no permission for deps.  Users just see the deps they have permission to see.